### DISCUSSION TITLE
Documentation Error in Batch mode `X.shape`

### DISCUSSION THREAD

**Discussion by JacobFV** [danielegrattarola/spektral#160] [General]
Hi. I was reading the documentation under "Data modes" and saw that the "batch mode" tensor shape for `X` is incorrectly described as `[batch, nodes, nodes]` (It should be `[batch, nodes, n_feat]`)
![Screenshot from 2021-01-10 19-51-46](https://user-images.githubusercontent.com/40343913/104141782-0fbd9700-537e-11eb-8886-224217a3dc7e.png)

**Comment by danielegrattarola** [comment]
You're correct, I've fixed this now. Thanks for pointing this out!
